src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
You can combine this, also `p` is not a very good name for variable. It's better to call `person` or smth 
```
const men = people.filter((person) => century 
  ? person.sex === 'm' && Math.ceil(person.died / 100) === century) 
  : person.sex === 'm'
```